charts/restate-helm/templates/service.yaml

```diff
@@ -10,7 +33,6 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
-  type: {{ .Values.service.type }}
```

Comment:
Why remove this? Maybe folks will want to configure a load balancer type service here.

Reply:
I'll add it back.